<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="idea-PC" name="com.scp.app.login.tests.Atest" tests="217" failures="0" timestamp="9 Mar 2019 13:59:21 GMT" time="0.349" errors="11">
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.015" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m5" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.Throwable" message="Method Atest.m5()[pri:0, instance:com.scp.app.login.tests.Atest@6767c1fc] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method Atest.m5()[pri:0, instance:com.scp.app.login.tests.Atest@6767c1fc] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m5 -->
  <testcase name="m9" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m1" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.031" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.015" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m8" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.015" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.031" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.002" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.015" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.001" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.015" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.015" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m2" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.002" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.015" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.017" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m6" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.016" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m4" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m3" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m7" time="0.000" classname="com.scp.app.login.tests.Atest">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at com.scp.app.login.tests.Atest.m7(Atest.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- m7 -->
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
  <testcase name="m9" time="0.000" classname="com.scp.app.login.tests.Atest"/>
</testsuite> <!-- com.scp.app.login.tests.Atest -->
